src/lib/statusEnums.ts

```diff
@@ -0,0 +1,9 @@
+export enum LoginStatus {
+    ALREADY_LOGGED_IN,
+    SUCCESSFUL_LOGIN,
+    SUCCESSFUL_LOGOUT
+}
+export enum RegisterStatus {
+    ALREADY_LOGGED_IN,
```

Comment:
Unterscheiden wir zwischen already logged in und logged in, weil dann entweder beide jetzt schon rein sollten oder nur logged in.